util/fiber_socket_base.h

```diff
@@ -107,15 +107,16 @@ class LinuxSocketBase : public FiberSocketBase {
   /// Creates a socket. By default with AF_INET family (2).
   error_code Create(unsigned short protocol_family = 2);
 
-  ABSL_MUST_USE_RESULT error_code Listen(const struct sockaddr* bind_addr, unsigned addr_len,
-                                         unsigned backlog);
+  ABSL_MUST_USE_RESULT error_code Bind(const struct sockaddr* bind_addr, unsigned addr_len);
+  ABSL_MUST_USE_RESULT error_code Chmod(const char* path, mode_t permissions);
```

Comment:
I actually do not think we need to export Chmod command. it's an implementation detail of `ListenUDS`. why not use chmod call directly?